css/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "mc-sgx-css"
+version = "0.1.0"
+authors = ["MobileCoin"]
+# See https://crates.io/category_slugs for valid categories
+categories = ["development-tools", "encoding"]
+description = "A brief summary of the crate"
```

:wrench: Should probably have a better description here. maybe:
```suggestion
description = "Enclave signature struct extractor"
```

---

I missed to change this. I'll add it